src/code/reducers/index.js

```diff
@@ -31,6 +31,63 @@ function initialState() {
           });
 }
 
+function fertilize(state) {
```

Comment:
Super helpful, thanks Sam - makes sense that `fertilize` is basically for breeding a very specific child from two parents, where we just need random children for clutch-breeding.